app/services/otp_rate_limiter.rb

```diff
@@ -47,15 +59,23 @@ def entry_for_current_phone
   end
   # rubocop:enable Naming/MemoizedInstanceVariableName
 
-  def otp_last_sent_at
-    entry_for_current_phone.otp_last_sent_at
-  end
-
   def otp_findtime
     IdentityConfig.store.otp_delivery_blocklist_findtime.minutes
   end
 
   def otp_maxretry_times
     IdentityConfig.store.otp_delivery_blocklist_maxretry
   end
+
+  def phone_fingerprint
+    @phone_fingerprint ||= Pii::Fingerprinter.fingerprint(phone)
```

Comment:
`OtpRequestsTracker` does `phone.strip` do we want to keep that here? or some sort of normalization?
```suggestion
    @phone_fingerprint ||= Pii::Fingerprinter.fingerprint(phone.strip)
```

Reply:
After noodling a bit, I think I'd rather do a PhoneFormatter on the Throttle-based values 🤔 (added in f34d940d7)